lib/cfg/cfg_Base.ml

```diff
@@ -0,0 +1,105 @@
+open Cfg_Node
+
+module CFG = struct
+  (* Auxiliary module for providing Hashtbl of a pretty printing function *)
+  module Hashtbl = struct
+    include Hashtbl
+
+    let pp pp_key pp_value ppf values =
+      Hashtbl.iter (fun key data -> Format.fprintf ppf "@[<1>%a: %a@]@." pp_key key pp_value data) values
+  end
+
+  type 'a item = {
+    idx   : int;
+    exp   : 'a;
+    pred  : int list;
+    succ  : int list;
+  } [@@deriving show]
+
+  (* The CFG is implemented as an Hashtable <id, (exp, predecessors, successors)> *)
```

Comment:
If I'm understanding the code correctly, this `id` is the same as the `idx` field of the following `item` definition, so why the two things have different names? This is confusing

The comment should be: (* The CFG is implemented as an Hashtable <idx, (idx, exp, predecessors, successors)> *)